core/platform/models.py

```diff
@@ -311,6 +311,22 @@ def import_cloud_translate_services(cls):
             from core.platform.cloud_translate import cloud_translate_services
             return cloud_translate_services
 
+    @classmethod
+    def import_logging_services(cls):
```

Comment:
This is actually quite weird because you're always importing "logging" and we don't have per-platform API services. I think this is not quite the right way to do it -- we should instead have an initialization phase where enable_logging() is called in a "starting script" (e.g. main.py) if we're not in emulator mode. That seems like a better way to model it.

Also I'm a bit confused by the dichotomy here. Why is enable_logging() disabled in emulator mode? (I don't see the link here -- usually emulator mode uses an explicit emulator.)

/cc @brianrodri